packages/fxa-graphql-api/src/lib/resolvers/types/payload/update-display-name.ts

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Field, ObjectType } from 'type-graphql';
+
+@ObjectType()
+export class UpdateDisplayNamePayload {
+  @Field({
+    description: 'A unique identifier for the client performing the mutation.',
+    nullable: true,
+  })
+  public clientMutationId?: string;
```

Comment:
I looked at that, and while GraphQL does support interfaces, they're only for data types like a query would return. Not for mutation inputs or payloads. There was some complex reason I found for this, as I actually was trying to set this up like that originally, before I ran into problems because that's not allowed.